Sources/Tracking/CustomerIO.swift

```diff
@@ -22,7 +24,18 @@ public protocol CustomerIOInstance: AutoMockable {
         jsonEncoder: JSONEncoder?,
         onComplete: @escaping (Result<Void, CustomerIOError>) -> Void
     )
-    func clearIdentify()
+
+    // sourcery:Name=screenView
+    func screen<RequestBody: Encodable>(
+        name: String,
+        // sourcery:Type=AnyEncodable
+        // sourcery:TypeCast="AnyEncodable(data)"
+        data: RequestBody?,
+        jsonEncoder: JSONEncoder?,
+        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void
+    )
+
+    func enableAutoScreenviewTracking()
```

Comment:
I imagined this being in the `CustomerIO.config {}` code, only. Is there a reason for there being a function here? 

Also, if there is a function to *enable* auto screen view tracking, should there also be a way to *disable* it, too?

Reply:
yeah this is more of a `setup` than an `enable` func, renamed